recipes/rcat/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "rcat" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jplind79/rcat/archive/v1.0.tar.gz
+  sha256: e91d57108c6032310c51537ad3e8bf8c5f99c66b0cefdf232cba73de4a1be141
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
There is no ``setup.py`` for this project. I think you will need to create one
Please see the docs for some instructions
https://packaging.python.org/tutorials/packaging-projects/

Reply:
Ok, thanks! Then I need to restructure the project a bit. I'll submit a new meta.yaml when I'm done.